src/UserStamping.php

```diff
@@ -2,11 +2,18 @@
 
 trait UserStamping {
 
+    /**
+     * Columns name.
+     *
+     * @var array
+     */
+    private $columns = [];
```

Comment:
Could this (even private) attribute name clash with others, I think so. Please update it with the trait name as a prefix.